.github/workflows/pull_requests.yml

```diff
@@ -111,9 +111,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: gaurav-nelson/github-action-markdown-link-check@v1
-        if: needs.docs-changed.outputs.any_changed == 'true'
         with:
           config-file: '.markdown_link_check.json'
+          use-quiet-mode: yes
+          check-modified-files-only: yes
+          base-branch: main
```

Comment:
What will happen on pull requests against other branches, e.g. `release-v2`?

Reply:
Not what we want. I'll change it to [github.base_ref.](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) Good catch!

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2720